test/spectests/spectests.cpp

```diff
@@ -179,6 +193,11 @@ class test_runner
                     pass();
                     continue;
                 }
+                catch (std::runtime_error const&)
```

Comment:
Hm isn't the goal to have every such case covered by `parser_error`?

Reply:
Ah yeah, this got here by accident, will remove.

LEB128 decoder throws `runtime_error` currently, I think it should be fixed to throw `parser_error` in another PR.

Reply:
Yes, I think changing that makes sense.